taskvine/src/worker/vine_process.c

```diff
@@ -600,4 +605,160 @@ int vine_process_measure_disk(struct vine_process *p, int max_time_on_measuremen
 	return result;
 }
 
+static int vine_process_sandbox_disk_measure(struct vine_process *p, int max_secs, struct path_disk_size_info **state)
```

Comment:
Since this function is similar to those in `dttools`, would it be better to add an extra argument to the `dttools` one with a list of paths to skip?

Reply:
I can give that a try

Reply:
I ended up duplicating the code in dttools since I felt there are enough usages of the function elsewhere where the extra arguments would not be relevant.